### DISCUSSION TITLE
FIFA 22 Controller problem

### DISCUSSION THREAD

**Discussion by ghost** [nefarius/DsHidMini#203] [Support]
Hey!

so I face a really annoying bug on FIFA 22 only, too much random pressing, and find it harder to go through other game modes of the game, please! Watch the video to take a deeper look at what I mean.

to be more precise. the gameplay experience was playable (inside the match, pass, shooting, etc.. everything went right) and the funny thing is, no random pressing at all, I was able to play smoothly and all controlled by me until the halftime menu and any menu of this game, I get the random pressing and scrolling again for no reason

to let you know, my PS3 controller is fully functional, it only does this on FIFA 22, but other games work properly with no random pressing or something like, I've tested around 6 games to make sure it does work properly (the video only had 3)

Link: https://www.youtube.com/watch?v=gJbmWhoSRHU

let me know your thoughts, thank you!

edit: I've tested GPJ and Xinput, still the same issue, 2 controllers on side select, and random pressing ...

**Comment by nefarius** [comment]
1. There are no "random" inputs, what you most probably experience is the game mistreating the controller layout
2. Have you tried DS4Windows mode? That works with all games with Xbox Controller support

**Comment by ghost** [comment]
Hey!
You were right! "mistreating the controller layout", on AC 3, only a few buttons working, also I've noticed this on DB Xenoverse 2 too, as it detects the controller and displays it by numbers, not the original layout ( A, X, Rb, etc..), majority of games I've tested so far has controller problems ... so I found a fix on youtube and it was soo simple, from now on, I have to launch games by "Big picture mode" on steam because it has a feature of controllers, it's like it loads original layout config in-game if launched by big picture mode

here are the settings I've used in order to fix that random pressing and mistreating controller layout, I don't mind using big picture mode, because it doesn't lead to any future problems

![FIX](https://user-images.githubusercontent.com/62818222/156528901-8c9f12e7-a0c2-4238-be65-bb27ecfbd5b0.png)

you can see it's like something loaded like config in-game, you'll know better than me in this
![wdqqwqwd](https://user-images.githubusercontent.com/62818222/156529886-6eb1f89f-70da-490d-967a-bb329a65c7ca.png)

if possible you can add this to the main Github page if anyone has already encountered this problem
Thanks and sorry for the late response :)

**Reply by nefarius** [reply]
Glad it works! I can't clutter the README with every single edge case, I think it's perfectly expectable from other users to invest 30 seconds into searching GH issues and discussions now that this case is documented 😉

**Comment by f-gozie** [comment]
Hey, so I'm having the same issues, and tried the fix you suggested that worked for you, but the issue still persists for me. Basically, even though I launch the game from steam, it still opens either the EA app or Origin (if I uninstall the EA app) and I don't see the "Config Loaded" popup that showed from you, so I guess the config doesn't actually get loaded, which makes the problem persist. Do you have any ideas around this?
